pkg/ccl/backupccl/backup_processor.go

```diff
@@ -284,13 +291,12 @@ func (bp *backupDataProcessor) ConsumerClosed() {
 }
 
 type spanAndTime struct {
-	// spanIdx is a unique identifier of this object.
-	spanIdx    int
-	span       roachpb.Span
-	firstKeyTS hlc.Timestamp
-	start, end hlc.Timestamp
-	attempts   int
-	lastTried  time.Time
+	span         roachpb.Span
+	firstKeyTS   hlc.Timestamp
+	start, end   hlc.Timestamp
+	attempts     int
+	lastTried    time.Time
+	finishesSpec bool
```

Comment:
nit: can we add a comment above this new field, its not immediately obvious how it influences progress reporting

Reply:
I'll get this in a followup instead of eating a ci for a comment.